extra/js/URLSearchParams.js

```diff
@@ -0,0 +1,45 @@
+function getURLParams() {
```

Comment:
We've been reserving the CapitalName convention for React components, maybe `url_parameters.js`?